doc/tutorials/realtime_performance_tuning.rst

```diff
@@ -77,78 +77,9 @@ Collecting Performance Monitoring Counters Data
 Enable Performance Monitoring Unit (PMU) Support in VM
```

Comment:
I think we can update this title too (since PMU support is now already enabled:
```suggestion
Performance Monitoring Unit (PMU) Support for the RTVM
```

Reply:
updated